Makefile

```
@@ -11,4 +11,4 @@ run:
 	foreman start
```

I bet we could remove foreman and the Procfile entirely and just inline this?
```suggestion
	bundle exec puma -p $PORT -S ~/puma -C config/puma.rb
```